### ISSUE TITLE
Uncaught TypeError: _this.notificationEl.querySelector is not a function

### ISSUE THREAD

**Issue by zamabraga** [ricardofbarros/linter-js-standard#187]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.17.0 x64
**Electron**: 1.3.15
**OS**: Ubuntu 16.04.2
**Thrown From**: [linter-js-standard](https://github.com/ricardofbarros/linter-js-standard) package 3.9.1

### Stack Trace

Uncaught TypeError: _this.notificationEl.querySelector is not a function

```
At /home/zama/.atom/packages/linter-js-standard/node_modules/atom-package-deps/lib/view.js:22

TypeError: _this.notificationEl.querySelector is not a function
    at /packages/linter-js-standard/node_modules/atom-package-deps/lib/view.js:22:58
```

### Commands

```
     -4:09.8.0 settings-view:open (atom-pane.pane.active)
  4x -3:19.1.0 core:backspace (input.hidden-input)
     -3:15.2.0 core:confirm (input.hidden-input)
  2x -2:04.2.0 core:paste (input.hidden-input)
     -1:19.8.0 command-palette:toggle (atom-workspace.workspace.scrollbars-visible-always.theme-one-dark-syntax.theme-one-dark-ui)
  4x -1:18.2.0 core:backspace (input.hidden-input)
     -1:13.8.0 core:move-down (input.hidden-input)
     -1:13.2.0 core:confirm (input.hidden-input)
     -1:13.2.0 settings-view:open (atom-workspace.workspace.scrollbars-visible-always.theme-one-dark-syntax.theme-one-dark-ui)
     -0:58.9.0 command-palette:toggle (atom-workspace.workspace.scrollbars-visible-always.theme-one-dark-syntax.theme-one-dark-ui)
  2x -0:57.3.0 core:move-down (input.hidden-input)
     -0:56.7.0 core:confirm (input.hidden-input)
     -0:56.7.0 sync-settings:restore (atom-workspace.workspace.scrollbars-visible-always.theme-one-dark-syntax.theme-one-dark-ui)
```

### Non-Core Packages

```
advanced-open-file 0.16.6 
atom-material-ui 2.0.3 
atom-terminal 0.8.0 
busy-signal 1.4.1 
docblockr 0.9.4 
git-plus 7.8.0 
intentions 1.1.2 
ionide-fake 1.2.2 
ionide-fsharp 1.9.3 
ionide-fsi 2.1.3 
ionide-installer 1.4.0 
ionide-paket 2.2.6 
ionide-webview 1.0.3 
language-docker 1.1.7 
language-groovy 0.7.0 
linter 2.1.4 
linter-js-standard 3.9.1 
sync-settings 0.8.1 
```

**Comment by Arcanemagus** [comment]
@ricardofbarros Your version of `atom-package-deps` is 2 major versions out of date, this should be fixed in the v4 series.

**Comment by natecox** [comment]
@Arcanemagus on a completely fresh install of Atom I get this same error, doubtful I have anything out of date at the system level. 

# Error Details

**Atom**: 1.17.0 x64
**Electron**: 1.3.15
**OS**: Mac OS X 10.12.5
**Thrown From**: [linter-js-standard](https://github.com/ricardofbarros/linter-js-standard) package 3.9.1


### Stack Trace

Uncaught TypeError: _this.notificationEl.querySelector is not a function

```
At /Users/nate/.atom/packages/linter-js-standard/node_modules/atom-package-deps/lib/view.js:22

TypeError: _this.notificationEl.querySelector is not a function
    at /packages/linter-js-standard/node_modules/atom-package-deps/lib/view.js:22:58
```

### Commands

```
```

### Non-Core Packages

```
firewatch-syntax 1.1.1 
linter-js-standard 3.9.1 
standard-formatter 2.8.0 
standardjs-snippets 2.6.0 
vim-mode 0.66.0 
```

**Comment by natecox** [comment]
Installing `linter` manually seems to have solved the issue

**Comment by Arcanemagus** [comment]
@natecox I tagged the package author, not you, for a reason 😉.

The [required version](https://github.com/ricardofbarros/linter-js-standard/blob/v3.9.1/package.json#L22) of `atom-package-deps` is currently listed as `^2.0.5`, but [that package](https://www.npmjs.com/package/atom-package-deps) is currently on v4.6.0.

**Comment by florianboudot** [comment]
@natecox Yes ! It worked for me. 
I closed atom, 
ran **`apm install linter-js-standard`** in the terminal,
reopened atom and it prompted to install a last dependency and it worked.
Thanks

**Comment by piersg** [comment]
@natecox @florianboudot NB I had no luck with `apm install linter-js-standard` by itself, but `apm install linter` (as per nate) did result in a prompt to finish from Atom next time I launched it. Hope @ricardofbarros comes back to us soon!

**Comment by dcposch** [comment]
cc @feross -- this issue makes standard hard to use with atom, sad times

**Comment by Lycanite** [comment]
I get this error but use Nuclide which replaces Linter.

**Comment by Arcanemagus** [comment]
@Lycanite This has been fixed on `master`, but is not yet released. Note that this problem has nothing to do with which consumer you are using, but this provider here 😉.

**Comment by Lycanite** [comment]
Ah I see, will wait for the release then, thanks!

**Comment by ernani** [comment]
Same here, is there any other workaround?

**Comment by sonicdoe** [comment]
The workaround for now is to manually install Linter using `apm install linter`. Other than that, I’ll release a new version including #193 soon.

**Comment by ernani** [comment]
Thanks a lot! That worked!

**Comment by sonicdoe** [comment]
Released in v4.0.0.
